src/vehicle.h

```diff
@@ -107,6 +107,7 @@ struct vehicle_part : public JsonSerializer, public JsonDeserializer
     friend vehicle;
     friend visitable<vehicle_cursor>;
     friend item_location;
+    friend class Pickup;
```

Comment:
Still working on that. Once I know exactly what `Pickup` needs access to I'll refactor it out via some acessors